src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/ETV2ContainerTest.groovy

```diff
@@ -35,4 +38,43 @@ class ETV2ContainerTest extends ETContainerTest {
                 .withLogConsumer(new Slf4jLogConsumer(LOGGER))
                 .waitingFor(Wait.forHttp("/api/v2/live"))
     }
+
+        def "Perform provide logs step with no reports"() {
+                given: "a test execution pipeline"
+                    String script = """
+                    node {
+                        withEnv(['ET_API_HOSTNAME=${etContainer.host}', 'ET_API_PORT=${etContainer.getMappedPort(ET_PORT)}']) {
+                            ttProvideLogs()
+                        }
+                    }
+                    """.stripIndent()
+                    WorkflowJob job = jenkins.createProject(WorkflowJob.class, "pipeline")
+                    job.setDefinition(new CpsFlowDefinition(script, true))
+                when: "scheduling a new build"
+                    WorkflowRun run = jenkins.buildAndAssertStatus(Result.SUCCESS, job)
+
+                then: "expect successful test completion"
+                    jenkins.assertLogContains("Providing ecu.test logs to jenkins.", run)
+                    jenkins.assertLogContains("[WARNING] No report files returned by ecu.test", run)
```

Comment:
This is a bit confusing for the user.He runs provideLogs and get a warning for report files. The user do not know that we need to ask ecu.test  for its report and i guess he/she does not care about it. Maybe an information like 
`No logfiles for this execution found.` with a hint that it could be that no test execution run before